src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -657,13 +658,21 @@ public DarCollection cancelDarCollectionElectionsAsChair(DarCollection collectio
    * @param collection The DarCollection
    * @return The updated DarCollection
    */
-  public DarCollection createElectionsForDarCollection(User user, DarCollection collection) {
+  public DarCollection createElectionsForDarCollection(User user, DarCollection collection)
+      throws Exception {
     try {
       List<String> createdElectionReferenceIds = collectionServiceDAO.createElectionsForDarCollection(
           user, collection);
-      List<User> voteUsers = voteDAO.findVoteUsersByElectionReferenceIdList(
-          createdElectionReferenceIds);
+      if (createdElectionReferenceIds == null || createdElectionReferenceIds.isEmpty()) {
```

Comment:
How can the result of `createElectionsForDarCollection()` ever be `null`? When calling an `internal` API that returns a collection type, we should be able to assert that we will never see `null`, only an empty collection.